README.md

```diff
@@ -172,6 +172,52 @@ app.use(g2o({
 }));
 ```
 
+### unauthenticatedStatusCode
+
+Optional; `Integer` (default: 401)
+
+If provided, the status code will be used as the response status code for when the authentication fails.
+
+### onUnauthenticated
+
+Optional; `Function` (default: noop)
+
+If provided, the function will be called after all checks have been completed and found to be unauthenticated.
```

Comment:
Great! And I realise now that I've been sending 401 back when I should really be sending 403. I'll fix that after this PR is merged.